modular_skyrat/modules/reagent_forging/code/forge.dm

```diff
@@ -128,641 +172,619 @@
 /obj/structure/reagent_forge/Initialize(mapload)
 	. = ..()
 	START_PROCESSING(SSobj, src)
+	populate_radial_choice_list()
 	update_appearance()
 
+/// Fills out the radial choice list with everything in the choice_list's contents
+/obj/structure/reagent_crafting_bench/proc/populate_radial_choice_list()
```

Comment:
this is `/obj/structure/reagent_crafting_bench`, others are `/obj/structure/reagent_forge`

Reply:
i knew i'd forget something